src/GlobalPayments.Api/Terminals/Builders/TerminalManageBuilder.cs

```diff
@@ -57,9 +93,10 @@ public class TerminalManageBuilder : TerminalBuilder<TerminalManageBuilder> {
         }
 
         protected override void SetupValidations() {
-            Validations.For(TransactionType.Capture).Check(() => TransactionId).IsNotNull();
+            Validations.For(TransactionType.Capture).When(() => AuthCode).IsNull().Check(() => TransactionId).IsNotNull();
             Validations.For(TransactionType.Void).When(() => ClientTransactionId).IsNull().Check(() => TransactionId).IsNotNull();
             Validations.For(PaymentMethodType.Gift).Check(() => Currency).IsNotNull();
+            Validations.For(TransactionType.Cancel).Check(() => Amount).IsNotNull();

```

Comment:
This cannot be here. As this breaks other terminal implementations that do not require an amount on a cancel command. I am looking at overhauling the validations so they can be more specific. We're not there yet, so this cannot be here.